webpack/react_app/components/TargetingHosts/TargetingHostsPage.js

```diff
@@ -39,9 +39,10 @@ const TargetingHostsPage = ({
     <br />
     <TargetingHosts apiStatus={apiStatus} items={items} />
     <Pagination
-      viewType="table"
       itemCount={totalHosts}
-      pagination={pagination}
+      page={pagination.page}
+      perPage={pagination.per_page}
+      perPageOptions={pagination.perPageOptions}
```

Comment:
If I am not mistaken, you don't need to pass those 3 props:
- page: is by default 1 in your implementation and is also default 1 in core, and will change based on the URL.
- perPage: I saw that you used `useForemanSettings();` which is also consumed in core Pagination
- perPageOptions: same as the above.

Reply:
I believe we don't so I'd agree with you here. But to keep the PR small can we keep this as is and do some clean-up in a followup PR? I believe there is a lot of code that is not needed now, but to shift to parameters will be a bit of an effort.